docs/source/deploying.rst

```diff
@@ -0,0 +1,46 @@
+Deploying
+=========
+After you've `setup your environment <getting-started.html>`_, deploying BinaryAlert is as easy as:
+
+.. code-block:: bash
+
+  $ ./manage.py deploy
+
+A ``deploy`` is equivalent to the following 4 operations executed in sequence:
+
+.. code-block:: bash
+
+  $ ./manage.py unit_test    # Unit tests ensure YARA rules compile correctly
+  $ ./manage.py build        # Build the Lambda ".zip" deployment packages
+  $ ./manage.py apply        # Runs "terraform apply" to update the infrastructure
+  $ ./manage.py analyze_all  # Starts a batch analysis of the entire S3 bucket
+
+.. warning:: To ensure new YARA rules are applied ASAP, *every* ``deploy`` starts a batch analysis. If this is undesired behavior, you can execute any of the aforementioned commands individually (e.g. only ``apply`` the changes).
```

Comment:
When would this be undesired behavior? (maybe provide an example or clarify)

Reply:
If there are multiple deploys in a short amount of time, or if you don't want to pay to re-analyze when all you did was change a setting. I'll clarify